source/simulator/solver_schemes.cc

```diff
@@ -906,6 +906,19 @@ namespace aspect
 
     return;
   }
+
+
+
+  template <int dim>
+  void Simulator<dim>::solve_no_advection_no_stokes ()
+  {
+    // This section is now empty but I guess I need some assertThrows?
+    // NOT SURE IF THIS IS NEEDED:
```

Comment:
Currently, I can not think of AssertThrows that are necessary. You would not get useful results with this scheme if you set end time to something else but 0, but who knows maybe someone has an application even for that, and I do not think it breaks anything.

And you need the line about the postprocessing, otherwise you will get no output if you set the `Run postprocessors on nonlinear iterations` parameter (if that is true, the solver scheme is responsible for calling the postprocessors).
Just remove the comment in capitals and this should be fine.

Reply:
Ok thanks. That is done. 

Procedural question: bellow saying the tester failed means the tester actually ran but failed or do you still have to manually ask the tester to run when all is ready?